sim/side.ts

```diff
@@ -342,7 +332,6 @@ export class Side {
 			return this.emitChoiceError(`Can't move: ${pokemon.name} can't use ${move.name} as a Z-move`);
 		}
 		if (zMove && this.choice.zMove) {
-			// TODO: The client shouldn't have sent this request in the first place.
```

Comment:
I've removed these `TODO`s - there's no reason a client wouldn't try to send these requests (in fact, my client has...). If the PS client is sending them then sure, its a bug on the PS client and let's fix it, but the simulator is meant to handle clients other than just the official client. Or am I missing something?

Reply:
I mean, it's a client TODO. Feel free to move it to client, though.